stopwatch.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+import * as watch from './stopwatch_func.js';
```

Comment:
Why not import * as stopwatch? And then rename the functions so you can call like

stopwatch.start
stopwatch.stop
stopwatch.reset
stopwatch.toggle

Way cleaner